### ISSUE TITLE
Can´t use ObjectMapper in WatchKit Extension: Linking against dylib not safe for use in application extensions

### ISSUE THREAD

**Issue by sticksen** [Hearst-DD/ObjectMapper#92]
I´m using `carthage` to add ObjectMapper to my WatchKit Extension.

...a warning occurs: 
`Linking against dylib not safe for use in application extensions`

![bildschirmfoto 2015-03-23 um 23 03 12 18 02](https://cloud.githubusercontent.com/assets/2657113/6778857/aee8e970-d156-11e4-890b-c279f85b41aa.png)

This is also described here: https://developer.apple.com/library/prerelease/ios/documentation/General/Conceptual/ExtensibilityPG/ExtensionScenarios.html

`To configure an app extension target to use an embedded framework, set the target’s “Require Only App-Extension-Safe API” build setting to Yes. If you don’t, Xcode reminds you to do so by displaying the warning “linking against dylib not safe for use in application extensions”.`

I´ve done this, the warning still occurs. Is this a problem on my side or does ObjectMapper need to do something? I´ve also included `Alamofire` and `SwiftyJSON` the same way, they link without warnings.

**Comment by ikesyo** [comment]
This setting was merged in ObjectMapper in #87, and the commit is not contained in the latest release v0.8. So you should specify `master` branch or commit after 4322310 in your Cartfile. Is it not the case?

**Comment by tristanhimmelman** [comment]
I've released a new version today (v0.9). Please pull the latest. This should fix your issue

**Comment by sticksen** [comment]
I´m using 0.9 now, but now the WatchKit Extension crashes:

```
Dyld Error Message:
  Library not loaded: @rpath/ObjectMapper.framework/ObjectMapper
  Referenced from: /private/var/mobile/Containers/Bundle/Application/80C4D949-FF36-4ED7-B417-265E2B62D7BE/FOCUS.app/PlugIns/WatchKit Extension.appex/WatchKit Extension
  Reason: no suitable image found.  Did find:
	/private/var/mobile/Containers/Bundle/Application/80C4D949-FF36-4ED7-B417-265E2B62D7BE/MyApp.app/PlugIns/WatchKit Extension.appex/Frameworks/ObjectMapper.framework/ObjectMapper: mmap() error 1 at address=0x1001E0000, size=0x00030000 segment=__TEXT in Segment::map() mapping /private/var/mobile/Containers/Bundle/Application/80C4D949-FF36-4ED7-B417-265E2B62D7BE/MyApp.app/PlugIns/WatchKit Extension.appex/Frameworks/ObjectMapper.framework/ObjectMapper
  Dyld Version: 353.10

Binary Images:
0x1200a0000 - 0x1200c7fff dyld arm64  <cbe9c4c5c080329aad62c7d2ef0e2b85> /usr/lib/dyld
0x19569c000 - 0x19578afff libxml2.2.dylib arm64  <4d23734e8cd6356bbbeb51968458abb6> /usr/lib/libxml2.2.dylib
```
